### DISCUSSION TITLE
Note map link depth

### DISCUSSION THREAD

**Discussion by wwwald** [TriliumNext/Trilium#2608] [Q&A]
I may be missing something in the documentation, but is there a way to limit the "link depth" of the note map?

Right now, running the 0.50.0-beta, I see a map like this for some of my most-used notes:
![image](https://user-images.githubusercontent.com/39531/152105884-1c209b17-14f3-4693-835d-66a379985db0.png)

Nice, but very hard to navigate or click through. I'd like to filter this to nodes only 1 or 2 links away from the current one. Is that possible?

**Comment by zadam** [comment]
No, currently there isn't a way to limit the depth.
